src/Microsoft.AspNetCore.Hosting/Startup/ExceptionPage/Views/ErrorPage.cs

```diff
@@ -0,0 +1,616 @@
+namespace Microsoft.AspNetCore.Hosting.Views
```

Comment:
Could you make the change to the generator to name this file ErrorPage.Generated.cs? If that suppresses diffs showing up in GitHub, that would be great.